src/key.rs

```diff
@@ -524,6 +524,7 @@ i8pcjGO+IZffvyZJVRWfVooBJmWWbPB1pueo3tx8w3+fcuzpxz+RLFKaPyqXO+dD
     }
 
     #[test]
+    #[ignore] // is too expensive
```

Comment:
is this a standard pattern to mark tests as "ignore"?

Reply:
yes